miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MessageToContact.kt

```diff
@@ -0,0 +1,17 @@
+package com.rakuten.tech.mobile.miniapp.js
+
+import androidx.annotation.Keep
+
+/** An object to prepare the message for sending to contacts. */
+@Keep
+data class MessageToContact(
+    val image: String,
+    val text: String,
+    val caption: String,
+    val action: String,
+    val title: String
```

Comment:
Looking at this now, the name `title` is misleading 🤔  `text` is actually the main text of the message and `title` is supposed to be the title of the Mini App (and `title` will be displayed underneath the main text in a smaller font).

@khairul-alam-licon @Climbatize @minh-rakuten  So maybe we should change this to either `miniAppTitle` or `subText`? What do you think?

Reply:
I see, to me `title` and `subText` are different concepts. If the purpose is to display this in a smaller font underneath the main text, then it can be `subText`.

Reply:
We only need to change `miniAppTitle`. The `text` is concise enough in Message object which implies the text content.